### ISSUE TITLE
Express links to old ontology term URLs

### ISSUE THREAD

**Issue by VladimirAlexiev** [Sveino/Inst4CIM-KG#70]
(Was "Use sameAs reasoning?")

@Sveino in https://github.com/Sveino/Inst4CIM-KG/issues/68
> use reasoning `<http://iec.ch/TC57/CIM100#> owl:sameAs <https://cim.ucaiug.io/ns#>`

- You can't declare `sameAs` between two namespaces (or rather, if you do, there'll be no effect)
- One should use:
  - `sameAs` for individuals
  - `equivalentClass` for classes
  - `equivalentProperty` for properties
- Their semantics is related but not the same:
  - `X sameAs Y` means that the repo needs to "copy" all statements `X P U` and `U P X` over to Y and vice versa
  - `X equivalentClass Y` means to "copy" all statements `U a X` as `U a Y` and vice versa
  - `X equivalentProperty Y` means to "copy" all statements `U X V` as `U Y V` and vice versa

But none of that is free.
- GraphDB implements owl:sameAs very efficiently (without copying) by recording statements against sameAs clusters of URLs, not individual URLs
  - But other repos may not implement `sameAs` reasoning efficiently
  - Some don't implement such reasoning at all, so you see queries like `X owl:sameAs* / prop Y`
  - It also has impact on counting
- Similar concerns apply to `equivalentClass, equivalentProperty`

So overall, I would recommend against it.
And whenever CIM namespaces change, to implement a database migration.

**Comment by VladimirAlexiev** [comment]
I said the same (but more briefly) in https://github.com/Sveino/Inst4CIM-KG/issues/123

**Comment by Sveino** [comment]
> So overall, I would recommend against it.
> And whenever CIM namespaces change, to implement a database migration.
I respectfully disagree. We need to use semantic description to describe the changes between version of the metadata as primary point. This must be done regardless of if we are doing sematic reasoning or database migration. When the semantic is clear, we can discuss if this should be solved though database migration or reasoning - but this is on the technical level and not the semantic layer.

**Comment by VladimirAlexiev** [comment]
We can add `dct:replaces` to each ontology term, pointing to the old term URL. Unlike `owl:equivalentProperty, equivalentClass`, this one is asymmetric so it captures the meaning.

We can also use `owl:priorVersion` to link the latest version of the ontology to a prior version. For this to be effective, we also need to
- modularize ontologies
- add `rdfs:isDefinedBy` to each term
- describe `vann:preferredNamespace` of each ontology
